conda/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "jina-hubble-sdk" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jina-hubble-sdk-{{ version }}.tar.gz
+  sha256: bab9b7b4d8c5965e099e07effd92cc0607b56eead0b4d26d6e5b473cb93daf8f
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools >=18.0
+    - wheel
+    - pip
+  run:
+    - python >=3.7
+    - requests ==2.27.1
+    - aiohttp ==3.8.1
+    - rich >=12.0.0
+
+test:
+  imports:
+    - hubble
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/jina-ai/hubble-client-python/
+  summary: SDK for Hubble API at Jina AI.
+  dev_url: https://github.com/jina-ai/hubble-client-python/
```

Comment:
```suggestion
  home: https://github.com/jina-ai/jina-hubble-sdk/
  summary: SDK for Hubble API at Jina AI.
  dev_url: https://github.com/jina-ai/jina-hubble-sdk/
```

Reply:
Maybe we should convert `hubble-client-python` to `jina-hubble-sdk`
 the following places in the file `setup.py `

[_project_urls ](https://github.com/jina-ai/jina-hubble-sdk/blob/main/setup.py#:~:text=%5D-,_project_urls,-%3D%20%7B)
[_download_url](https://github.com/jina-ai/jina-hubble-sdk/blob/main/setup.py#:~:text=hubble%2Dclient%2Dpython/%27-,_download_url,-%3D%20%27https%3A//github)

[_url](https://github.com/jina-ai/jina-hubble-sdk/blob/main/setup.py#:~:text=)-,_url,-%3D%20%27https%3A//github)